sources/packages/api/src-sql/Types/Update-coe-status.sql

```diff
@@ -0,0 +1,18 @@
+--Removing the value Not Required from sims.coe_status.
+--As postgres does not support removal of an Enum Value, We create a temporary enum and rename it.
+CREATE TYPE sims.coe_status_to_rollback AS ENUM ('Required', 'Completed', 'Declined');
+
+ALTER TABLE
```

Comment:
Do we need this _ALTER TABLE_? _coe_status_ is a new field so there will not be any conflict while executing the migrations or even rolling them back, right?
Are the migration being executed in a order where the _coe_status_ is added to the disbursement table and then the _coe_status_ enum is updated?

Reply:
@andrewsignori-aot Yes, we need the ALTER table statement. Otherwise, DROP TYPE sims.coe_status is failing. The coe_status column in disbursement_schedule needs to be assigned to temporary enum before we drop sims.coe_status.

_Are the migration being executed in a order where the coe_status is added to the disbursement table and then the coe_status enum is updated?_ - Yes the coe_status is added in PR Part 1 itself. That is why we needed the above alter script.